math/mean/mean.go

```diff
@@ -0,0 +1,27 @@
+package mean
+
+import (
+	"fmt"
+)
+
+func mean(values []float64) float64{
```

Comment:
Please capitalise this function, otherwise how would you expect it to work outside this package.

Reply:
Just edited the function name. Please review.